README.md

```diff
@@ -478,6 +485,10 @@ Work in progress drivers. Help the authors make these crates awesome!
 [APDS9960]: https://crates.io/crates/apds9960
 [SSD1675]: https://crates.io/crates/ssd1675
 [CC1101]: https://crates.io/crates/cc1101
+[MPU9250_2]: https://crates.io/crates/mpu9250
```

Comment:
Have you made an effort to join forces with the author of the other driver? Not a big fan of confusing people with 2 identical drivers; if this is an evolution of the original driver then I'd prefer replacing the old entry.

CC @japaric

Reply:
published mpu9250 has more features by now and also battle-tested. japaric/mpu9250 was WIP and wasn't published. So the right solution is to delete japaric/mpu9250 from the list, as it's outdated and causes people to do things like https://github.com/JoshMcguigan/mpu9250/commits/master %)

Reply:
I agree but the right thing to do is to ask @japaric whether it's okay for him, too, before replacing the previous entry.

Reply:
Sure!

Reply:
Since this is a maintained, updated fork I'm OK with removing my old version.

Reply:
Thanks!

Reply:
Lets remove the old version in this same PR then, please?

Reply:
Make a separate commit removing it and add a note reference to @japaric 's agreement.